ha_cluster_exporter.go

```diff
@@ -149,7 +151,7 @@ func resetClusterMetrics() {
 	err := prometheus.Register(nodeResources)
 	if err != nil {
 		log.Println("[ERROR]: failed to register NodeResource metric. Perhaps another exporter is already running?")
-		log.Panic(err)
+		log.Println(err)
 	}
```

Comment:
maybe we should return this error instead, and use the logger in the caller (`main()`)?